pkg/release/release.go

```diff
@@ -22,6 +24,7 @@ type Config struct {
 	ApprClient *apprclient.Client
 	HelmClient *helmclient.Client
 	Logger     micrologger.Logger
+	RestConfig *rest.Config
```

Comment:
IMO this is a setting and not a dependency.

Reply:
Done.